plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/reload/SpringBeanReload.java

```diff
@@ -236,7 +237,7 @@ private void doReload() {
                 continue;
             }
             //beanDefinition enhanced: BeanFactoryPostProcessor
-            ProxyReplacer.clearAllProxies();
+            ProxyReplacer.clearProxiesByNames(beanNamesForProxyReplacer);
```

Comment:
I think maybe these clear action should be moved to destroy method of DefaultListableFactory. Because any bean is destroyed should clear the cache.

Reply:
Okay, I'll see how to change it later.